lib/logstash/inputs/lumberjack.rb

```diff
@@ -29,19 +29,33 @@ class LogStash::Inputs::Lumberjack < LogStash::Inputs::Base
   # SSL key passphrase to use.
   config :ssl_key_passphrase, :validate => :password
 
+  # Number of maximum clients that the lumberjack input will accept
```

Comment:
@ph can you add why we are immediately deprecating this setting in the comments. Will be useful for anybody wondering...